src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -242,15 +234,6 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
     setSellAsset(defaultSellAsset)
   }, [defaultSellAsset, sellAsset, setSellAsset, walletConnectedChainIds])
 
-  useEffect(() => {
-    // WARNING: do not remove.
-    // clear the confirmed quote on mount to prevent stale data affecting the selectors
-    dispatch(tradeQuoteSlice.actions.resetConfirmedQuote())
-    // clear the active quote index on mount to prevent stale data affecting the selectors
-    dispatch(tradeQuoteSlice.actions.resetActiveQuote())
-    dispatch(tradeInput.actions.setSlippagePreferencePercentage(undefined))
-  }, [dispatch])
-
```

Comment:
Note the above removal. All of the routes in the parent component `MultiHopTrade` benefit from persistent store state across mounts, but we DO clear both the `tradeQuoteSlice` and  `tradeInputSlice` completely when `MultiHopTrade` is mounted, so we're safe.